sql/migrations/20200225214500_world.sql

```diff
@@ -0,0 +1,22 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200225214500');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200225214500');
+-- Add your query below.
+
+-- Correct Shimmering Gloves enchantment id to match Classic data: https://classic.wowhead.com/item=6565/shimmering-gloves
+UPDATE `item_template` SET `random_property`='710' WHERE `entry`='6565';
```

Comment:
The random property used by this item is already 710. Template data for all items was updated based on the blizzcon classic demo back in 2018.